app/helpers/alert_helper.rb

```diff
@@ -7,10 +7,15 @@ def alert_tag(flash)
   end
 
   def alert_content_for(type, message)
-    content_tag(:div, class: "alert alert-#{type}") do
+    content_tag(:div, class: "alert alert-#{type}", role: 'alert') do
       concat(content_tag(:p, message, id: "flash_#{type}", class: 'alert-message'))
-      concat(content_tag(:a, class: 'alert-close', data: { dismiss: 'alert' }) do
-        content_tag :i, nil, class: 'fa fa-times-circle fa-2x'
+      concat(content_tag(:button,
+        type: 'button',
+        class: 'alert-close',
+        data: { dismiss: 'alert' },
+        aria: { label: 'Close alert' }
+      ) do
+        content_tag(:i, nil, class: 'fa fa-times-circle fa-2x', 'aria-hidden': 'true')
       end)
```

Comment:
Im not sure if Im testing this properly but the alert that appears when you try to favorite something when you are not logged in is not read by the screen reader, I think probably I don't know how to test it.

Reply:
@anna-dorigo Just in case, I expanded the js on this pop-down to ensure the screen reader announces the label on it correctly!